recipe_scrapers/_utils.py

```diff
@@ -73,14 +74,28 @@ def get_yields(element):
     return servings
 
 
-def normalize_string(string):
-    # Convert all named and numeric character references (e.g. &gt;, &#62;)
-    unescaped_string = html.unescape(string)
+def normalize_string(string, strip_html=True):
+    # The goal here is to get this string as clean as possible. Address the following cases
+    #  - Remove leading/trailing whitespace
+    #  - Remove escaped whitespace characters from whole string (e.g \n, \t)
+    #  - Remove HTML Tags
+    #  - Convert all named and numeric character references (e.g. &gt;, &#62;)
+
+    # Deal with HTML and HTML Encoded Characters
+    string = strip_tags(
+        f"<tag>{string}<tag>"
+    )  # This is a workaround, since HTMLParser expects valid markup
+    string = strip_tags(
+        f"<tag>{string}<tag>"
+    )  # This is another workaround, handles "&amp;amp;"
```

Comment:
Strange - what's the output for this example input?  (and should we add a test case to cover the workaround/situation?)

Reply:
Yeah, I found this very strange as well. By chance, the tests we currently have cover these unique situations. I found them after introducing `strip_tags` because they started to fail.

For `&amp;&amp;` consider this example (derived from food52 tests)
```
from recipe_scrapers._utils import normalize_string

text = 'Sticky Pomegranate &amp;amp; Black Pepper Chicken Wing'
print(normalize_string(text))
>>> Sticky Pomegranate & Black Pepper Chicken Wing

Reply:
Hmm, I did just check what it did with 3 `&amp;&amp;&amp;` and the normalized string was

```
Sticky Pomegranate &amp; Black Pepper Chicken Wing

Reply:
Maybe this is a bug with the underlying `HTMLParser`.